src/plugins/links/public/components/editor/links_panel_editor_link.tsx

```diff
@@ -23,14 +23,14 @@ import {
 } from '@elastic/eui';
 import { DashboardContainer } from '@kbn/dashboard-plugin/public/dashboard_container';
 
-import { NavigationLinkInfo } from '../../embeddable/types';
+import { LinksLinkInfo } from '../../embeddable/types';
 import { validateUrl } from '../external_link/external_link_tools';
 import { fetchDashboard } from '../dashboard_link/dashboard_link_tools';
-import { NavEmbeddableStrings } from '../navigation_embeddable_strings';
+import { LinksStrings } from '../links_strings';
 import { DashboardLinkStrings } from '../dashboard_link/dashboard_link_strings';
-import { DASHBOARD_LINK_TYPE, NavigationEmbeddableLink } from '../../../common/content_management';
+import { DASHBOARD_LINK_TYPE, LinksLink } from '../../../common/content_management';
 
-export const NavigationEmbeddablePanelEditorLink = ({
+export const LinksPanelEditorLink = ({
```

Comment:
```suggestion
export const LinksEditorLink = ({
```
+ rename file to `links_editor_link.tsx`